architecture-model-generator/modules/architecture-model-generator-core/src/main/java/io/ballerina/designmodelgenerator/core/CommonUtils.java

```diff
@@ -137,4 +144,73 @@ public static ModuleInfo from(ModuleID moduleId) {
                     moduleId.version());
         }
     }
+
+    /**
+     * Returns the resource path string for the given resource method symbol.
+     *
+     * @param semanticModel the semantic model
+     * @param resourceMethodSymbol the resource method symbol
+     *
+     * @return the resource path string
+     */
+    public static String getResourcePathStr(SemanticModel semanticModel,
+                                            ResourceMethodSymbol resourceMethodSymbol) {
+
+        io.ballerina.modelgenerator.commons.ModuleInfo moduleInfo;
```

Comment:
Can we remove the full classpath, and add an import instead.


```suggestion
        ModuleInfo moduleInfo;
```

Reply:
ModuleInfo record exist in this scope